x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -218,3 +218,29 @@ export const getBenchmarkFilterQuery = (
     : '';
   return baseQuery + sectionQuery + ruleNumberQuery;
 };
+
+export const getBenchmarkFilterQueryV2 = (
```

Comment:
looks like we get the same params and return a string, why do we need a new version?

Reply:
previously we only accept string for the rule number and section filter, but now since we add ability to filter based on multiple rule number and/or section we also accept Array